articles/multifactor-authentication/developer/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Developer Documentation for Guardian
+url: /multifactor-authentication/developer
+description: Developer Documentation for Guardian
+topics:
+  - mfa
+  - guardian
+contentType:
+  - index
+useCase:
+  - customize-mfa
+---
+# Developer Resources for Guardian
+
+At Auth0, we believe in giving developers the tools they need to get their jobs done. With Guardian, developers are able to customize their users' experience, and even build their own applications on top of our multi-factor capabilities.
```

Comment:
~At Auth0, we believe in giving developers the tools they need to get their jobs done.~ With Guardian, developers are able to customize their users' experience, and even build their own applications on top of our multi-factor capabilities.